src/App.js

```diff
@@ -1,8 +1,25 @@
-import React from 'react'
+import React, { Component } from 'react'
+import { BrowserRouter, Route, Switch } from 'react-router-dom'
 
-import { PostList } from 'components/PostList/PostList'
+import Home from './views/Home/Home'
+import About from './views/About/About'
 
-//eslint-disable-next-line
-import styles from './App.module.scss'
+import Header from './components/Header/Header'
 
-export const App = () => <PostList />
+import './css/app.general.scss'
+
+class App extends Component {
+  render() {
+    return (
+      <BrowserRouter>
+        <Header />
+        <Switch>
+          <Route path="/" component={Home} exact />
+          <Route path="/about" component={About} />
+        </Switch>
+      </BrowserRouter>
+    )
+  }
+}
+
+export default App
```

Comment:
Do not use 'export default'. In this project we will use only named exports.
example: 
```js
export const App = () => ...
```